backend/hdfs/fs.go

```diff
@@ -58,7 +60,7 @@ func NewFs(ctx context.Context, name, root string, m configmap.Mapper) (fs.Fs, e
 		return f, fs.ErrorIsFile
 	}
 
-	//fs.Config.LogLevel.Set("DEBUG")
+	//fs.GetConfig(ctx).LogLevel.Set("DEBUG")
```

Comment:
Nick will ask you to remove dead code

Reply:
Fixed by commit [51f23ac](https://github.com/rclone/rclone/pull/4640/commits/51f23ac1bb97c894c985a8e6b1bc8cd89ef03b36)